backend/functions/sms.ts

```diff
@@ -0,0 +1,60 @@
+//Created Twilio Number, authToken, accountSid; Currently on free trial.
+require('dotenv').config()
+const twilio = require('twilio');
+const accountSid = process.env.TWILIO_ACCOUNT_SID;
+const authToken = process.env.TWILIO_AUTH_TOKEN;
+const client = require('twilio')(accountSid, authToken);
+const twilioNumber = process.env.TWILIO_PHONE_NUMBER;
+
+
+//TESTING WITH STANDALONE FUNCTION
+// var numbersToMessage = ["+17036384616"]
+
+// numbersToMessage.forEach(function (number) {
+//   var message = client.messages.create({
+//     body: 'OKB Testing Message - Pranav',
+//     from: twilioNumber,
+//     to: number
+//   })
+//     .then(message => console.log(message.status))
+// });
```

Comment:
Delete this code if you've already confirmed it works (if you're making further modifications based on my suggestions it's fine to keep this, but once you're going to merge please delete)